app/javascript/controllers/autosubmit_controller.ts

```diff
@@ -75,12 +76,19 @@ export class AutosubmitController extends ApplicationController {
   }
 
   private preventAutosubmit(
-    target: HTMLElement & { disabled?: boolean },
+    target: HTMLElement & { disabled?: boolean } & { value?: string },
```

Comment:
marrant que t'es pas fait : 
```suggestion
    target: HTMLElement & { disabled?: boolean; value?: string },
```

mais en vrai osef